values-gcp.yaml

```diff
@@ -0,0 +1,3 @@
+storageClass: nifi-storage-class
```

Comment:
This is a much smarter way to handle this, to put it into its own values-gcp.yaml file. Didn't realize we could use multiple -f arguments with helm. Probably makes sense to rename the storageClass to something like "gce-ssd" though?

Reply:
that was originally my intent with the naming, but if it's being pulled out into a file called "values-gcp.yaml" then that seems weird to me... perhaps we should just rename it to values-pvc.yaml or something? It seems odd to name it 'nifi-storage-class' with the intent of that being generic and then put into a values file named after google cloud platform

Reply:
Maybe we could use another variable like `customStorageClass: true/false` and then we can name the storageClass whatever we wan't?

Reply:
I've just pushed it